src/icalendar/tests/test_parsing.py

```diff
@@ -1,3 +1,19 @@
+'''Tests checking that parsing works'''
+import pytest
+
+from icalendar.parser import Contentline, Parameters
+
+@pytest.mark.parametrize('raw_content_line, expected_output', [
+    # Issue #142 - Multivalued parameters. This is needed for VCard 3.0.
+    # see https://github.com/collective/icalendar/pull/142
+    ('TEL;TYPE=HOME,VOICE:000000000', ('TEL', Parameters({'TYPE': ['HOME', 'VOICE']}), '000000000')),
+    # Issue #143 - Allow dots in property names. Another vCard related issue.
+    # see https://github.com/collective/icalendar/pull/143
+    ('ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR:;;This is the Adress 08; Some City;;12345;Germany', ('ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR', Parameters(), ';;This is the Adress 08; Some City;;12345;Germany'))
```

Comment:
Is there a missing test for folded input e.g. `'ITEMADRNULLTHISISTHEADRESS08158SOMECITY12345.ADR:;;This is the Adress 08; \r\n Some City;;12345;Germany'`

Reply:
Is this an addition? If yes, please create an issue or a PR to solve this. :heart: